AppVeyor failures

AppVeyor currently fails to build any PRs, log:
```
cmake.exe : CMake Error at CMakeLists.txt:21 (project):
At C:\projects\icinga2\tools\win32\configure.ps1:37 char:1
+ & cmake.exe .. `
+ ~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (CMake Error at ...t:21 (project)::String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
  The CMAKE_C_COMPILER:
    C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.12.25827/bin/Hostx86/x64/cl.exe
  is not a full path to an existing compiler tool.
CMake Error at CMakeLists.txt:21 (project):
  The CMAKE_CXX_COMPILER:
    C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.12.25827/bin/Hostx86/x64/cl.exe
  is not a full path to an existing compiler tool.
```

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -15,7 +15,7 @@ branches:
     - master
 
 cache:
-- build
+- build -> appveyor.yml
 - vendor -> tools\win32\download-openssl.ps1
 - C:\ProgramData\chocolatey\lib\winflexbison3
 

